src/Helper/HeadLink.php

```diff
@@ -405,6 +405,7 @@ public function createDataStylesheet(array $args)
             }
         }
 
+        $extras = [];
```

Comment:
I'm sorry, but can you move this to line 386. Thanks!

Reply:
I have move the other $extra variable too to the default groups. please check